File: lexilogicserver/src/main/java/org/amalgam/lexilogicserver/model/utilities/referenceobjects/Player.java
package org.amalgam.lexilogicserver.model.utilities.referenceobjects;

public class Player {
    private int userID;
    private String username;
    private String password;
    private String lastLoggedIn;

    public Player(int userID, String username, String password, String lastLoggedIn) {
        this.userID = userID;
        this.username = username;
        this.password = password;
        this.lastLoggedIn = lastLoggedIn;
    }

    public int getUserID() {
        return userID;
    }

    public void setUserID(int userID) {
        this.userID = userID;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getLastLoggedIn() {
        return lastLoggedIn;
    }

    public void setLastLoggedIn(String lastLoggedIn) {
        this.lastLoggedIn = lastLoggedIn;
    }
}

File: Client_Python/Utilities/Reference_Objects/Player.py
class Player:
    def __init__(self, userID, username, password, lastLoggedIn):
        self.userID = userID
        self.username = username
        self.password = password
        self.lastLoggedIn = lastLoggedIn

    @property
    def userID(self):
        return self._userID

    @userID.setter
    def userID(self, userID):
        self._userID = userID

    @property
    def username(self):
        return self._username

    @username.setter
    def username(self, username):
        self._username = username

    @property
    def password(self):
        return self._password

    @password.setter
    def password(self, password):
        self._password = password

    @property
    def lastLoggedIn(self):
        return self._lastLoggedIn

    @lastLoggedIn.setter
    def lastLoggedIn(self, lastLoggedIn):
        self._lastLoggedIn = lastLoggedIn


